lib/coin_flip_demo.dart

```diff
@@ -14,6 +11,8 @@ class CoinFlipDemoState extends State<CoinFlipDemo> with SingleTickerProviderSta
   AnimationController _controller;
   Animation _flipAnimation;
 
+  static final Random random = Random();
```

Comment:
nit: make a member instead of static not that it matters

Reply:
done.